lapis2/src/test/kotlin/org/genspectrum/lapis/silo/SiloQueryTest.kt

```diff
@@ -361,6 +361,16 @@ class SiloQueryTest {
                 }
                 """,
                 ),
+                Arguments.of(
+                    StringEquals("theColumn", null),
+                    """
+                {
+                    "type": "StringEquals",
+                    "column": "theColumn",
+                    "value": null
+                }
```

Comment:
```suggestion
                    {
                        "type": "StringEquals",
                        "column": "theColumn",
                        "value": null
                    }
```

Reply:
(Actually also the indent of most other JSONs here is wrong - at least for my taste)